api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -775,28 +775,30 @@
           <type>String</type>
         </field>
         <field>
-          <!-- TODO: replaced by <Source> with "main" scope and "resources" language -->
           <name>resources</name>
           <version>3.0.0+</version>
           <description>
             This element describes all the classpath resources such as properties
             files associated with a project. These resources are often included in the final
             package.
             The default value is {@code src/main/resources}.
+
+            @deprecated Replaced by &lt;Source&gt; with "main" scope and "resources" language.
```

Comment:
```suggestion
            @deprecated Replaced by {@code &lt;Source&gt;} with "main" scope and "resources" language.
```

Reply:
`{@code]` escapes HTML characters, so the output would really be `&lt;Source&gt;` in the Javadoc. If it was a Java file, I would have written {@code <Source>} (allowing direct use of `<` and `>` are the purpose of `{@code]` and `{@literal]`). But since this is a `.mdo` file, I wasn't sure if `<Source>` wouldn't be interpreted as XML. I thought that escaping the XML characters was the safest, which means that I cannot put them in a`{@code]` element.

Reply:
Ah sorry, I jsut saw the CI output. I see that `&lt;` has been replaced by `<` in the generated Java file. Then you are right, I need to add the `{@code}` tags.